src/main/java/com/webdev/cosmo/cosmobackend/emulator/FacebookNotifEmulator.java

```diff
@@ -0,0 +1,4 @@
+package com.webdev.cosmo.cosmobackend.emulator;
+
+public class FacebookNotifEmulator {
```

Comment:
Is this class used anywhere?

Reply:
Will be, but making code latetly does not help haha. I guess it can stay as I already have it feveloped locally and will lublish it today or in next days